SpatialGDK/Source/SpatialGDK/Public/EngineClasses/AbstractVirtualWorkerTranslator.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+
+class SPATIALGDK_API AbstractVirtualWorkerTranslator
+{
+public:
+	virtual ~AbstractVirtualWorkerTranslator() {};
+	virtual VirtualWorkerId GetLocalVirtualWorkerId() const PURE_VIRTUAL(AbstractVirtualWorkerTranslator::GetLocalVirtualWorkerId, return SpatialConstants::INVALID_VIRTUAL_WORKER_ID;);
```

Comment:
Random thought. This could also be done with a protected memeber if we wanted to de-virtualise the interface.

Reply:
We could but that likely won't be sufficient if we want to expand the set of mocked functions to more involved functions but haven't yet mocked things like the receiver / worker connection. I'd rather leave like this.